### DISCUSSION TITLE
Folders inside the Media Player Browser

### DISCUSSION THREAD

**Discussion by MiAutomations** [fwestenberg/reolink_dev#326] [Q&A]
Hello 

After the installation of the Reolink dev I notice that I have this folders created by the devices camera working on this integrations/addon 

![image](https://user-images.githubusercontent.com/47602733/138109033-0e54a405-3c27-451a-aaee-856785aa2055.png)

How can I regular delete this folders and the content of this folders ? I don't  see this folders in the config file over the samba access... 

Thank you and best regards

**Comment by johnnysmithy852456** [comment]
I had a similar question some months back, I wanted to know where these files where located and I was told that they are in the camera's sd card.
HA is just showing you the icons and the content, I think it's FTP
When you view the video, HA steams the data from your camera to the video player in HA
So to answer your question I would imagine you would set it up in the cameras setting for how long you keep the files ￼
